packages/jest-core/src/TestScheduler.ts

```diff
@@ -336,12 +336,12 @@ class TestScheduler {
   async _setupReporters() {
     const {collectCoverage: coverage, notify, verbose} = this._globalConfig;
     const reporters = this._globalConfig.reporters || [['default', {}]];
-    let summary = false;
+    let summaryOptions;
 
     for (const [reporter, options] of reporters) {
       switch (reporter) {
         case 'default':
-          summary = true;
+          summaryOptions = options;
```

Comment:
I went checking if `options` is always defined. Apparently normalization always add an object. That’s smart idea!